oauthdoctor/bin/gen-bin.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+root=<Enter your Google Ads Doctor root path here>
+bin=$root/bin
+
+# Remove previously generated binaries
+rm -f $root/windows
+rm -f $root/osx
+rm -f $root/linux
+
+# Generate binaries from master branch
+pushd $root
```

Comment:
pushd and popd are not guaranteed to be available in bash. You should explicitly cd. You can save the current directory into an variable with ${PWD}.